src/main/java/ru/yandex/qatools/ashot/cropper/ImageCropper.java

```diff
@@ -4,14 +4,13 @@
 import ru.yandex.qatools.ashot.coordinates.Coords;
 
 import java.awt.image.BufferedImage;
-import java.io.Serializable;
 import java.util.Set;
 
 /**
  * @author <a href="pazone@yandex-team.ru">Pavel Zorin</a>
  */
 
-public abstract class ImageCropper implements Serializable {
```

Comment:
Yes, ofc.. I'll change it, but could you please describe the usage use-case (when it should be serializable)?